webpack/config/loaders.js

```diff
@@ -57,7 +57,7 @@ module.exports = props => {
       ]
     },
     {
-      test: /\.(png|jpeg|jpg|gif|bmp|tif|tiff|svg|woff|woff2|eot|ttf)$/,
+      test: /\.(png|jpeg|jpg|gif|bmp|tif|tiff|svg|woff|woff2|eot|ttf|JPG)$/,
```

Comment:
I believe the `i` flag at the end of a regex pattern makes it case insensitive, so theoretically this may be a more robust match pattern:

```js
/\.(png|jpeg|jpg|gif|bmp|tif|tiff|svg|woff|woff2|eot|ttf)$/i
```

@MarcioPorto can you test that out and see if it still works? If so, let's use that instead.